agent/bin/kontena-agent

```diff
@@ -31,6 +31,8 @@ Kontena::Logging.initialize_logger(STDOUT, log_level)
 
 agent = Kontena::Agent.new(
   api_uri: api_uri,
-  api_token: api_token
+  api_token: api_token,
+  ssl_verify: ENV['KONTENA_SSL_VERIFY'],
+  ssl_ca: ENV['KONTENA_SSL_CA'],
```

Comment:
Don't necessarily need the `KONTENA_SSL_CA`, because openssl itself understands `SSL_CERT_DIR=` and `SSL_CERT_FILE=` when using the default cert store?